_data/applications.yml

```diff
@@ -0,0 +1,159 @@
+# official: nil (unofficial; non-team RH developed) second (this is omission of `official`),
+# official: false (third party) last
+
+# The Cockpit group is special, it inherits source, issues, license for the `source: cockpit` entries
+cockpit:
+  source: https://github.com/cockpit-project/cockpit/
+  issues: https://github.com/cockpit-project/cockpit/issues
+  license: LGPL 2.1
+  comment: |
+    This is the default template that "source: cockpit" pulls from, so source, issues, and license don't need to be filled out everywhere.
+
+machines:
+  title: Machines
+  source: https://github.com/cockpit-project/cockpit-machines
+  issues: https://github.com/cockpit-project/cockpit-machines/issues
+  package: cockpit-machines
+  official: true
+  license: LGPL 2.1
+  description: |
+    Create, run, and manage virtual machines in Cockpit.
+
+podman:
+  title: Podman Containers
+  source: https://github.com/cockpit-project/cockpit-podman
+  issues: https://github.com/cockpit-project/cockpit-podman/issues
+  official: true
+  license: LGPL 2.1
+  description: |
+    Download, use, and manage containers in your browser. (Podman replaces Docker.)
+
+selinux:
+  title: SELinux
+  official: true
+  source: cockpit
+  package: cockpit-selinux
+  description: |
+    View and manage SELinux exceptions inside Cockpit. Instant apply overrides and export your rules to apply to other systems.
+
+storaged:
+  title: Storage
+  official: true
+  source: cockpit
+  package: cockpit-storaged
+  description: |
+    Manage your system's storage. Supports local partitions, encryption, NFS, RAID, iSCSI, and more.
+
+kdump:
+  title: Kernel Dump
+  official: true
+  source: cockpit
+  package: cockpit-kdump
+  description: |
+    Debugging the Linux kernel? Cockpit's kdump add-on helps catch stack traces.
+
+networkmanager:
+  title: Network
+  official: true
+  source: cockpit
+  package: cockpit-networkmanager
+  description: |
+    Manage your network interfaces and edit your firewall with ease.
+
+packagekit:
+  title: Software Updates (PackageKit)
+  official: true
+  source: cockpit
+  package: cockpit-packagekit
+  distro: Fedora, Red Hat Enterprise Linux, Debian, Ubuntu
+  description: |
+    See and apply updates to your system. Supports RPM and DEB based systems through PackageKit.
+
+ostree:
+  title: Software Updates (OSTree)
+  official: true
+  package: cockpit-ostree
+  source: https://github.com/cockpit-project/cockpit-ostree
+  issues: https://github.com/cockpit-project/cockpit-ostree/issues
+  license: LGPL 2.1
+  description: |
+    Upgrade OSTree-based systems in your browser.
+
+composer:
+  title: Cockpit Composer
+  package: cockpit-composer
+  source: https://github.com/osbuild/cockpit-composer/
+  issues: https://github.com/osbuild/cockpit-composer/issues
+  description: |
+     Composer generates custom images suitable for deploying systems or uploading to the cloud. It integrates into Cockpit as a frontend for osbuild composer.
```

Comment:
It created only rpm based images, shall we be more specific here?

Reply:
Uh...? It creates images for VMs, not RPMs, right? I thought I copied this one though.

Reply:
Yep, copy/pasted from https://github.com/osbuild/cockpit-composer/#cockpit-composer